### PR TITLE
Content

### LINKED ISSUE #6: Draft Website Content
Choose at least 3 things to showcase on your website and write out whatever text description you'd like to have for them.

Write this in a markdown file. Work on a seperate branch, and create a pull request when you're ready.

### PR DESCRIPTION
**Pull request by hvk5388**
includes the content description

### CODE REVIEW

File: description.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Zoom-bot
+The Zoom-bot is an interactive navigation and maintence interface that could be implemented on subways and trains all over the world to improve riders experience.
+The Zoom-bot is an interface to be placed on trains and subways. It should assist users by cleaning the train and giving them information including directions and air quality. The Zoom-bot will also keep track of train cleanliness and alert maintenance workers of problems. Subway employees will be using the interface to track trains’ cleanliness and location. 
+#Clinique Virtual Reality
+Clinique Virtual Reality is a way to experience personalized cosmetic advice from any location, at any time. It allows the user to scan their face, view the problem areas, and recommends products and solutions. 
+#UXPA
+As the founding treasurer of UXPA, Penn State Chapter, I have put in a lot of work to get the club started. UXPA strives to give UX students oportunities to gain expreience outside of the classroom. We haven't currently finished any projects, but we are well on our way.
```
Comment (mjkern) [inline description.md]: The first two sound pretty good, but the last one does not quite measure up. I would not say anything about projects if you have not done any, but I would replace "put in a lot of work" with a list of your specific responsiblities or things you have done. Maybe consider flipping the first two sentances.

File: description.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#Zoom-bot
+The Zoom-bot is an interactive navigation and maintence interface that could be implemented on subways and trains all over the world to improve riders experience.
+The Zoom-bot is an interface to be placed on trains and subways. It should assist users by cleaning the train and giving them information including directions and air quality. The Zoom-bot will also keep track of train cleanliness and alert maintenance workers of problems. Subway employees will be using the interface to track trains’ cleanliness and location. 
+#Clinique Virtual Reality
+Clinique Virtual Reality is a way to experience personalized cosmetic advice from any location, at any time. It allows the user to scan their face, view the problem areas, and recommends products and solutions.
```
Comment (mjkern) [inline description.md]: Nice work on the first two. My only question is what you actually created/delivered. Did you build full applications? Or just wireframes? Did you user test? Did you collaborate or work individually?

**Review by mjkern — CHANGES_REQUESTED** [review]
Nice work Hannah, I made some suggestions but ultimately it is your website so need to address them.

This pr should not contain your html though, please make sure it is saved on another branch and then remove it.

**Comment by mjkern** [comment]
@hvk5388 Have you addressed these concerns yet?

**Comment by hvk5388** [comment]
yes i pushed the changes

File: description.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+The Zoom-bot is an interactive navigation and maintence interface that could be implemented on subways and trains all over the world to improve riders experience. It should assist users by cleaning the train and giving them information including directions and air quality. The Zoom-bot will also keep track of train cleanliness and alert maintenance workers of problems. Subway employees will be using the interface to track trains’ cleanliness and location. The high fidelity wireframes have been developed. 
+#Clinique Virtual Reality
+Clinique Virtual Reality is a way to experience personalized cosmetic advice from any location, at any time. It allows the user to scan their face, view the problem areas, and recommends products and solutions. I worked on creating high fiedlity wireframes for the application.
+#UXPA
+ UXPA strives to give UX students oportunities to gain expreience outside of the classroom.As the founding treasurer of UXPA, Penn State Chapter, I have put in a lot of work to get the club started. Leading meetings, recruiting members, and workign to become a recognized club are some of my tasks. We haven't currently finished any projects, but we are well on our way.
```
Comment (mjkern) [inline description.md]:
```suggestion
 UXPA strives to give UX students oportunities to gain expreience outside of the classroom. As the founding treasurer of UXPA, Penn State Chapter, I have put in a lot of work to get the club started. Leading meetings, recruiting members, and working to become a recognized club are some of my tasks. We haven't currently finished any projects, but we are well on our way.
```

**Review by mjkern — CHANGES_REQUESTED** [review]
Thanks for updating those. I'd still love to hear Mom's thoughts here, but I made my last comment below.

**Review by mjkern — APPROVED** [review]
Nice work!

**Pull request merged**